website/docs/docs/collaborate/column-level-lineage.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Column level lineage"
+description: "dbt Explorer provides recommendations that you can take to improve the quality of your dbt project."
+---
+
+dbt Explorer now offers column level lineage (CLL) for the resources in your dbt project. Analytics engineers can quickly and easily gain insight into the provenance of their data products at a more granular level. For each column in a dbt resource (model, source, or snapshot), Explorer provides the full lineage for the data in that column.
+
+Column level lineage is available to dbt Cloud Enterprise accounts that can use Explorer. It’s also available through the Discovery API.
+
+:::tip Beta
+Column-level lineage is now available in beta. Check it out! We'd love to [know what you think](https://docs.google.com/forms/d/e/1FAIpQLSdpCbVkGY9QwfExFonpWE4DTOKi3fQxBGLD0wwKYpkMjgcE7g/viewform)!
+:::
+
+## No setup required
+
+There is no additional setup required for column level lineage if your account is on an Enterprise plan that can use Explorer. You can access column level lineage by expanding the column card in the **Columns** tab of an Explorer [resource details page](/docs/collaborate/explore-projects#view-resource-details) for a model, source, or snapshot.
+
+Lineage will update after each run executed in the production environment. Make sure that `docs generate` is running within at least one job in the environment. Refer to [Generating metadata](/docs/collaborate/explore-projects#generate-metadata) for more details.
+
+<Lightbox src="/img/docs/collaborate/dbt-explorer/example-cll.png" title="Example of the Columns tab and where to expand for the CLL"/>
+
+<LoomVideo id='278c948ba387457884cc6b9545793685' />
+
+## What you can use column level lineage for {#use-cases} 
+
+Click on these tabs to learn more about the CLL use cases, the analysis you can do, and the results you can achieve:
```

Comment:
Explore the sections below to learn more about why and how you can use column-level lineage.

Reply:
updated this. for accessibility, "below" is not a great word choice so worked around that